src/prognostic_equations/surface_temp.jl

```diff
@@ -41,19 +100,27 @@ function surface_temp_tendency!(Yₜ, Y, p, t, slab::PrognosticSurfaceTemperatur
         Q = FT(0)
     end
 
-    @. Yₜ.sfc.T -=
-        (sfc_rad_e_flux + sfc_turb_e_flux + Q) /
-        (ρ_ocean * cp_ocean * depth_ocean)
+    # TODO: The convention of "precipitation" being positive *upward* is very confusing. 
```

Comment:
I thought the idea was to use the same convention for the sign of fluxes in the entire code base. Converting them somewhere at the surface in Atmos may be more prone to errors